include/meltpooldg/heat/laser.hpp

```diff
@@ -14,6 +14,12 @@ namespace MeltPoolDG::Heat
 {
   using namespace dealii;
 
+  enum LaserImpactType
```

Comment:
Could you use 
```suggestion
  enum class LaserImpactType
```
to not potentially spoil the namespace?